sources/platform/console/authentication.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Authentication
+description: Learn about Apify Console's account creation and authentication process.
+sidebar_position: 5
+category: platform
+slug: /console/authentication
+---
+
+**Learn about Apify Console's account creation and authentication process.**
+
+---
+
+## Sign up
```

Comment:
Do we need the section on how to sign up? It is already covered in the main doc about Apify Console? Since this is much more expanded look at hot to sign up maybe it could be moved there instead of being here with doc that should be focused only on two factor authentication?

Reply:
It doesn't make much sense to me to remove it from index.md as it should serve as introduction to Apify and by extension Apify Console. I would much rather move the signing up from authentication.md to index.md and leave authentication.md as solely authentication focused document.